### ISSUE TITLE
Dotnet core 3.1, crash on windows 7

### ISSUE THREAD

**Issue by kinsung** [SteveSandersonMS/WebWindow#83]
Dotnet core 3.1, crash on windows 7.

**Comment by jinqi166** [comment]
see:[https://blog.stevensanderson.com/2019/11/18/2019-11-18-webwindow-a-cross-platform-webview-for-dotnet-core/](url)
How it works
On Windows, WebWindow uses the new Chromium-based Edge via webview2, assuming you have that browser installed (it could fall back on older Edge if you don’t, but I haven’t implemented that)
On Mac, it uses the OS’s built-in WKWebView, which is the same technology behind Safari
On Linux, it uses WebKitGTK+2, which is yet again a WebKit-based technology

**Comment by kinsung** [comment]
The new Chromium-based Edge is installed, and it crash when creating web view com instance.

**Comment by Y56380X** [comment]
Is Microsoft Visual C++ Redistributable for Visual Studio 2019 installed on your System?

**Comment by kinsung** [comment]
> Is Microsoft Visual C++ Redistributable for Visual Studio 2019 installed on your System?
Yes, the application is  crashed when creating web view instance via com interface.
Thank you for your reply.

**Comment by Progmasys** [comment]
@kinsung if you still insterested, You could try removing this method GetDpiForWindow(_hWnd) since it is Win10 only. Or add some loginc like this:

unsigned int WebWindow::GetScreenDpi()
{
if (IsWindows10OrGreater)
{
return GetDpiForWindow(_hWnd);
}
else
{
return 0;
}
}

And you probably would need to remove this too: SetThreadDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE);

This changes worked for me on Win7 x64.
